hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/NetworkAdminServiceConfig.java

```diff
@@ -27,5 +28,5 @@
  */
 @ConfigData("networkAdmin")
 public record NetworkAdminServiceConfig(
-        @ConfigProperty(defaultValue = "/opt/hgcapp/services-hedera/HapiApp2.0/data/upgrade/current")
+        @ConfigProperty(defaultValue = "/opt/hgcapp/services-hedera/HapiApp2.0/data/upgrade/current") @NetworkProperty
```

Comment:
This is a local file path.  I would expect that it needs to be node specific.

Reply:
Absolutely, this should be a node property.

Reply:
Changed it to node property.